src/backend/PublicInterfaceImpl.c

```diff
@@ -239,45 +236,17 @@ int BSL_API_ApplySecurity(const BSL_LibCtx_t *bsl, BSL_SecurityResponseSet_t *re
             if (conclusion == BSL_SECOP_CONCLUSION_SUCCESS)
             {
                 BSL_LOG_DEBUG("Security operation success, target block num = %" PRIu64, sec_oper->target_block_num);
-                continue;
             }
-
-            BSL_PolicyAction_e err_action_code = sec_oper->failure_code;
```

Comment:
This is removing the action being taken based on this value, but is there a need to actually store the value in the secop? Is it used solely by the PP and not by the SC? That is okay, but should be documented on the struct member that it is not intended to be used by SC.